parlai/chat_service/services/messenger/agents.py

```diff
@@ -56,23 +56,21 @@ def observe_typing_on(self, persona_id=None):
         """
         self.manager.message_sender.typing_on(self.id, persona_id=persona_id)
 
-    def put_data(self, message):
+    def put_data(self, messenger_message):
         """
         Put data into the message queue if it hasn't already been seen.
         """
-        mid = message['message']['mid']
-        seq = message['message'].get('seq', None)
-        if 'text' not in message['message']:
-            print(
-                'Msg: {} could not be extracted to text format'.format(
-                    message['message']
-                )
-            )
-            return
-        text = message['message'].get('text')
-        if text is None:
-            text = message['message']['payload']
-        img_attempt = True if 'image' in message['message'] else False
+        mid = messenger_message['message']['mid']
+        seq = messenger_message['message'].get('seq', None)
+        message = messenger_message['message']
```

Comment:
message vs. messenger_message is slightly confusing, although i know what you mean by it. perhaps we could come up with a better name here, although i can't think of one off the top of my head

Reply:
it's honestly incredibly confusing, i figured if i included `messenger` in there it would be more clear that the message is coming from `messenger`; perhaps we can call it `messenger_data`?

Reply:
yeah messenger data definitely sounds better